wolfcrypt/test/test.c

```diff
@@ -682,7 +682,7 @@ THREAD_RETURN WOLFSSL_THREAD wolfcrypt_test(void* args)
 int wolfcrypt_test(void* args)
 #endif
 {
-    int ret;
+    int ret = 0, errors_are_fatal = 0;
```

Comment:
How does the errors_are_fatal variable get set?

Reply:
Manually by editing the source code :)

that's the only place where it's set ... could also be a define or `static const int`.

Reply:
I've changed it to be modifiable by either defining `WOLFSSL_CRYPT_TEST_CONTINUE_ON_ERROR` on compile time and for the interactive tests (with arguments) by passing the `-c` option.